.gitignore

```diff
@@ -7,3 +7,7 @@ coverage/
 npm-debug.log
 .idea/
 .jshintrc
+
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
it should be just `.vscode` not the content of `.vscode/settings.json`

As you can see below, the file still appear in your push